azavea/pwd-unitybar
# Create dist bundle

## Overview

This PR adds infrastructure for generating distribution bundles on releases and adjusts the release instructions to note that one should generate the dist bundles and check them in on releases.

Connects #25

### Notes

- `babel-polyfill` apparently can't be nested, so I change this back to use `transform-runtime` as Babel recommends for libraries
- to reduce the size of the bundles, I made `react` and `react-dom` into externals which are then loaded from the apps which use this. This works pretty seamlessly except that the `react-onclickoutside` dependency expects ReactDOM 16.x to work correctly now. This upgrade will have to happen in the apps this gets used in, though.

## Testing Instructions
- run `.scripts/cibuild` and verify that it outputs two bundles in the `dist` dir: one minified, one not. Output should look similar to: https://travis-ci.org/azavea/pwd-unitybar/builds/308162185
- check the testing instructions in the GARP/EPA STAR cognate PR.
